### PR TITLE
Fix static tests

### PR DESCRIPTION
**Pull request by brentschmaltz**
Fixes a static test issue.

### CODE REVIEW

File: build/dependenciesTest.props [hunk 1]
```diff
@@ -7,6 +7,15 @@
...
     <XunitVersion>2.4.0</XunitVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' != 'net461'">
     <SystemTextJson>8.0.4</SystemTextJson>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
+    <SystemTextEncodingsWeb>4.7.2</SystemTextEncodingsWeb>
+    <SystemTextJson>4.7.2</SystemTextJson>
```
Comment (jennyf19) [inline build/dependenciesTest.props:18]: i htink we have to move everything to 8.0.4 for the CVE

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: build/dependenciesTest.props (hunk 1 above)
Comment (brentschmaltz) [reply build/dependenciesTest.props:18]: This is just for 461.

File: test/Microsoft.IdentityModel.Tokens.Tests/ClaimsIdentityFactoryTests.cs [hunk 2]
```diff
@@ -41,8 +43,6 @@ public void Create_FromTokenValidationParameters_ReturnsCorrectClaimsIdentity(bo
             {
                 Assert.IsType<ClaimsIdentity>(actualClaimsIdentity);
             }
-
-            AppContextSwitches.ResetAllSwitches();
```
Comment (pmaytak) [inline test/Microsoft.IdentityModel.Tokens.Tests/ClaimsIdentityFactoryTests.cs]: I would keep this at the end of both classes?

**Review by pmaytak — APPROVED** [review]

**Review by pmaytak — APPROVED** [review]

**Pull request merged**